client/app/queue/substituteAppellant/tasks/utils.js

```
@@ -60,7 +60,7 @@ export const closedTasksToHide = [...automatedTasks, ...nonAutomatedTasksToHide,
 // This may be refined after user testing...
 export const openTasksToHide = [...nonAutomatedTasksToHide, ...automatedTasks];
 // active tasks that can be cancelled by the user during a same appeal substitution.
-export const openTasksToShow = [...mailTasks, ...hearingAdminActions];
+export const openTasksToShow = [...mailTasks, ...hearingAdminActions, 'TranslationTask'];
```

Hm.  Is there a way to make this work for `TranslationTask` without hardcoding in `TranslationTask`?  I think hardcoding can get a bit brittle.

---

Sorry, I'm just catching up... is there a special case around `TranslationTask`?

---

Hey, good question.  

There isn't a special case around `TranslationTask`.  When I did my first review, I locally tested with an appeal that had a `TranslationTask` and unselected that task.  However, the `TranslationTask` did not show up as cancelled on the appeal.  Part of why I think it would be better to avoid hardcoding `TranslationTask` is because even though I saw this bug with `TranslationTask`, I can't think of anything particular to `TranslationTask` that made the bug appear in that scenario.  That just happened to be the task type that I selected.

I hope the above was clear.....

---

Why do we need `openTasksToShow`? The default should be to show any open tasks unless we specifically hide certain types (hence `openTasksToHide`), right?

---

refactored.